docs/whats-new/release-candidate-1.md

```diff
@@ -0,0 +1,786 @@
+---
+title: .NET Aspire 9.0 (Release Candidate 1)
+description: Learn what's new with .NET Aspire 9.0 (Release Candidate 1).
+ms.date: 10/11/2024
+---
+
+# .NET Aspire 9.0 (Release Candidate 1)
+
+.NET Aspire 9.0 is the next major release of .NET Aspire; it will support BOTH .NET 8 (LTS) and .NET 9. .NET Aspire 9.0 addresses some of the most highly requested features and pain points from the community. 
+
+## Upgrade to .NET Aspire 9.0
+
+Instructions on how to upgrade here
+
+- [Install .NET Aspire 9.0 templates](#templates)
+- [Upgrade an existing project manually](#upgrade-an-existing-project-to-net-aspire-90-manually)
+- [Upgrade an existing project using the Upgrade Assistant](#upgrade-an-existing-project-to-net-aspire-90-using-the-upgrade-assistant)
+
+### Acquisition
+
+.NET Aspire 9.0 simplifies acquisition by providing a .NET Aspire SDK available as the [📦 `Aspire.AppHost.Sdk`](https://nuget.org/packages/Aspire.AppHost.Sdk) NuGet package. This will no longer require an extra step to acquire the tooling required to build .NET Aspire projects. In other words, starting with .NET Aspire 9—you don't need to install a .NET workload.
```

Comment:
I wonder if we can define "acquisition" in terms of what the user's goals.

```suggestion
.NET Aspire 9.0 makes it simpler to configure your environment to develop Aspire applications. You no longer need a .NET workload. Instead, add the [📦 `Aspire.AppHost.Sdk`](https://nuget.org/packages/Aspire.AppHost.Sdk) MSBuild SDK to your project and it will be downloaded via NuGet automatically.
```